psychopy/experiment/components/apriltag/__init__.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Part of the PsychoPy library
+# Copyright (C) 2002-2018 Jonathan Peirce (C) 2019-2021 Open Science Tools Ltd.
+# Distributed under the terms of the GNU General Public License (GPL).
+
+from __future__ import absolute_import, print_function
+
+from contextlib import contextmanager
+from os import path
+from pathlib import Path
+from psychopy.experiment.components import image
+from psychopy.experiment.components.image import ImageComponent, Param, getInitVals
+from psychopy.localization import _translate
+
+
+class AprilTagComponent(ImageComponent):
+    """An event class for presenting image-based AprilTag stimuli"""
+
+    categories = ['Stimuli']
+    targets = ['PsychoPy', 'PsychoJS']
+    iconFile = Path(__file__).parent / 'apriltag.png'
+    tooltip = _translate('AprilTag: present AprilTag image')
+
+    _apriltag_images_dir = Path(__file__).parent / 'apriltag_images'
```

Comment:
Should there be a folder under experiment->components->apriltag? There's nothing in the repo so I just get "could not find file" - does it need adding to git or is it something users would need to manually add?